### DISCUSSION TITLE
where can change the transaction timeout，  5000ms too short

### DISCUSSION THREAD

**Discussion by baiheinet** [teableio/teable#830] [Q&A]
![image](https://github.com/user-attachments/assets/47c316be-c73d-4cf1-899f-7aba9f66c31e)

**Comment by tea-artist** [comment]
What specific operation did this error occur during?

**Reply by baiheinet** [reply]
Pasted data automatically associated,just over 500 row

**Reply by tea-artist** [reply]
We are currently using Prisma's default transaction time and do not provide an env configuration for the time being. According to our test, 5000 rows pasting can be completed within 5s under the resource of 2c 4g
